src/core/ActionMenu/ActionMenuItem/ActionMenuItem.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { ReactNode, AriaRole } from 'react';
+import classnames from 'classnames';
+import { HtmlLi } from '../../../reset';
+import { SuomifiThemeConsumer, SuomifiThemeProp } from '../../theme';
+import { baseStyles } from './ActionMenuItem.baseStyles';
+import styled from 'styled-components';
+import { Icon, BaseIconKeys, IconProps } from '../../Icon/Icon';
+import { RouterLink } from '../../Link';
+import {
+  ActionMenuProviderState,
+  ActionMenuConsumer,
+} from './../ActionMenuPopover/ActionMenuPopover';
+
+export interface ActionMenuItemProps {
+  /** Custom class */
+  className?: string;
+  /** Text of the action */
+  children: ReactNode;
+  /** Disables the item */
+  disabled?: boolean;
+  /**
+   * Icon from suomifi-theme
```

Comment:
Minor thing, but I'd unify these descriptions to be like above if they fit single line 😁

Reply:
Fixed